modules/prometheus-exporter_oracle/conf/05-alertlog.yaml

```
@@ -0,0 +1,18 @@
+module: oracle
+name: "Alert.log errors count"
+id: AlertLogError
+
+transformation: true
```

```suggestion
transformation: true
aggregation: true
``` 

here you do not force aggregation at all, can you not do the same for other detectors ?

---

Yes, i will do and drop the aggregation.

---

ok but you still need to create the transformation variable

---

ok , this means I have to add "aggregation: true" on it, and for all detectors ?

---

except if you think that give the ability to user to change aggregation is risky regarding the detector behavior.

here I think it is safe the let the user customize this (for example to get an alert title with a subset of dimensions instead of all of them which could be a bit long as you noticed before)